lib/utils.js

```diff
@@ -4,20 +4,33 @@
 'use strict';
 
 const database = require('bedrock-mongodb');
-const {Ed25519KeyPair} = require('crypto-ld');
+const {LDKeyPair} = require('crypto-ld');
+const v1 = require('did-veres-one');
+const didKey = require('did-method-key');
 
 exports.getCollection = collectionName => database.collections[collectionName];
 
-exports.setKeyId = async ({key, method = 'key'} = {}) => {
+/**
+ * @param {AsymmetricKey} key - WebKMS AsymmetricKey.
+ * @param {string} [didMethod] - DID method to use for key id.
+ * @param {string} [didMethodMode] - Relevant for v1 type dids, testnet or live.
+ * @returns {Promise}
+ */
+exports.setKeyId = async ({key, didMethod, didMethodMode} = {}) => {
```

Comment:
Maybe the whole problem here is that this function is trying to do too many things. It's trying to compute a key ID for the key *and* setting it. Perhaps it should just be `computeKeyId` and then we can replace where it's used to be:

```js
key.id = await computeKeyId({...});
```